pkg/sbom/spdx/marshal.go

```diff
@@ -184,6 +193,9 @@ func (m *Marshaler) Marshal(ctx context.Context, bom *core.BOM) (*spdx.Document,
 			relationShips = append(relationShips, m.spdxRelationShip(refA, refB, m.spdxRelationshipType(rel.Type)))
 		}
 	}
+
+	bom.Vulnerabilities()
```

Comment:
What is this for?

Reply:
hmm... Maybe I was checking something and forgot to delete it.
Thanks. Removed in https://github.com/aquasecurity/trivy/pull/7213/commits/b1ca52e3eb5698984b4015d0a3219474af3ac7f9